files/external_node_v2.rb

```diff
@@ -376,6 +382,7 @@ def watch_and_send_facts(parallel)
           cache(certname, result)
         end
       rescue TimeoutError, SocketError, Errno::EHOSTUNREACH, Errno::ECONNREFUSED, FactUploadError
+        Net::HTTPServerException
```

Comment:
This doesn't catch the error, it's on a new line with no comma on the preceding line. Currently it simply fails with a message stating there's a 500 erver error.